packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -108,53 +108,55 @@ const JurisdictionCard: React.FC<TJurisdictionCardProps> = ({ isAdded, isSelecte
                             {header}
                         </WalletText>
                     </div>
-                    {rows.map(row => {
-                        return (
-                            <JurisdictionCardRow
-                                description={parseDescription(row)}
-                                key={`wallets-jurisdiction-card--${row?.title}`}
-                                renderTag={() => {
-                                    if (!row?.titleIndicators) return;
+                    <div className='wallets-jurisdiction-card-rows'>
+                        {rows.map(row => {
+                            return (
+                                <JurisdictionCardRow
+                                    description={parseDescription(row)}
+                                    key={`wallets-jurisdiction-card--${row?.title}`}
+                                    renderTag={() => {
+                                        if (!row?.titleIndicators) return;
```

Comment:
minor/optional: given comlexity and size, does contents of renderTag deserve its own component?

Reply:
Yes @wojciech-deriv, i do plan to refactor jurisdiction soon.